attributes/default.rb

```diff
@@ -4,18 +4,23 @@
 #
 # Copyright 2014-2016, Bloomberg Finance L.P.
 #
+::Chef::Node.send(:include, ConsulCookbook::Helpers)
```

Comment:
They're not, unfortunately. I needed this to utilize the helper methods. I'll be rebasing shortly.